src/main/kotlin/com/doomspork/helloworld/HelloWorldConfiguration.kt

```diff
@@ -5,5 +5,15 @@ import org.hibernate.validator.constraints.NotEmpty
 import com.fasterxml.jackson.annotation.JsonProperty
 import com.fasterxml.jackson.annotation.JsonCreator
 
-data class HelloWorldConfiguration [JsonCreator] (JsonProperty("template") NotEmpty var template: String = "",
-                                   JsonProperty("defaultName") NotEmpty var defaultName: String = "Stranger") : Configuration()
+data class HelloWorldConfiguration() : Configuration() {
```

Comment:
How about something like this, which passes the tests and is still concise:

```kotlin
data class HelloWorldConfiguration [JsonCreator] (JsonProperty("template") NotEmpty var template: String,
                                                  JsonProperty("defaultName") NotEmpty var defaultName: String) : Configuration()
```

It looks like removing the assignment does the trick, what do you think?